raiden/blockchain_events_handler.py

```diff
@@ -327,16 +327,16 @@ def handle_channel_closed(raiden: "RaidenService", event: Event):
         else:
             # Must be a light client
             latest_non_closing_balance_proof = LightClientMessageHandler.get_latest_light_client_non_closing_balance_proof(
-                channel_state.identifier, channel_state.partner_state.address, raiden.wal.storage
+                channel_state.identifier, channel_state.our_state.address, raiden.wal.storage
```

Comment:
im a little bit concern about how this was working before for LC - FN scenarios

i mean, this change could break the different topologies? 

- LC - FN
- LC - FN - LC (same hub)
- LC - FN -LC (different hub)